plugins/CoreUpdater/Controller.php

```diff
@@ -179,8 +180,24 @@ public function oneClickUpdatePartTwo()
 
         $messages = [];
 
+        $nonce = Common::getRequestVar('nonce', '', 'string');
+        if (empty($nonce)) {
+            return json_encode(['no token']);
+        }
+        $value = Option::get('NonceOneClickUpdatePartTwo');
```

Comment:
not using a constant here as depending how they upgrade or code changes maybe it wouldn't exist. could use one though

Reply:
👍